### ISSUE TITLE
Compatibility with AU inverters

### ISSUE THREAD

**Issue by Uskompuf** [drobtravels/solaredge-local#14]
Should this work with Australian inverters with setapp support?

https://www.solaredge.com/sites/default/files/se-single-phase-HD-wave-inverter-setapp-datasheet-aus.pdf

Thanks

**Comment by drobtravels** [comment]
I'm _guessing_ it would, but no way to know for sure until someone tests it.  The API is unpublished.

Do you have the inverter already?  If so, give it a shot and let us know please!

**Comment by dragoshenron** [comment]
Yes, this wrapper will work also with aussie inverters like the one linked. You can close the issue
